packages/rlc-common/src/metadata/packageJson/packageCommon.ts

```diff
@@ -43,7 +43,7 @@ export function getCommonPackageDevDependencies(
     "@microsoft/api-extractor": "^7.40.3",
     "@types/node": "^18.0.0",
     eslint: "^8.55.0",
-    prettier: "^3.2.5",
+    prettier: "^3.3.3",
```

Comment:
I wonder if we really rely on prettier in non azure?

Reply:
I did a search and found that there are some @msinternal tests which use "prettier" but also reference `../../../.prettierrc.json` so I don't know what the design there. For example, https://github.com/Azure/autorest.typescript/blob/1de92fdbe7ed4b830621c85bef6e56d91b06fd47/packages/typespec-ts/test/integration/generated/type/model/flatten/package.json#L48

I didn't see "prettier" being used in other test baselines, so maybe it is safe to remove them, unless we also want to have the ability to format after a non-azure package is generated.

Reply:
Since the standalone packages still invoke "prettier" even though the config file path is definitely not valid if not within azure-sdk-for-js.  I logged a new issue to discuss further: https://github.com/Azure/autorest.typescript/issues/2700

Reply:
I removed all "prettier" dependencies and formatting scripts from non azure monorepo code generation.